### DISCUSSION TITLE
Looking to open website on google chrome and make some changes to pictures

### DISCUSSION THREAD

**Discussion by jc2400** [DustinBrett/daedalOS#454] [Q&A]
So I am pretty new to anything html and java in general but I found your website really interesting and was hoping to get it running off of vscode in order to work through the code and make slight changes to the pictures and icon photos to personalize and understand how the code actually works. After cloning the repository from github I understand that I needed to download extensions in order for the code to be executed. I guess my question is how can I open the website through google chrome to see how small changes (such as the url name, wallpaper, options that show up for the right click menu) affect how the website would show up on a browser. Other than that think the project is super cool and am looking forward to any updates in the future!

**Comment by DustinBrett** [comment]
You should not need any extensions for the code to execute. The steps to get the code running in VS Code are listed here (https://github.com/DustinBrett/daedalOS?tab=readme-ov-file#development).

You need to have node.js & yarn installed. Then you can just run `yarn install`, `yarn build:prebuild` & `yarn dev` to start a dev session. The console will say once it's started and which site to go (http://localhost:3000/) and view it.

I have suggested VS Code extensions (https://github.com/DustinBrett/daedalOS/blob/main/.vscode/extensions.json) you can install to make development easier, but they should not be required.
